modules/vm/templates/entrypoint.sh.tpl

```diff
@@ -165,7 +165,42 @@ fi
 
 # TODO: -Xmx based on the machine's memory size
 
-# TODO: Backup cron
+# Configure the GraphDB backup cron job
+
+cat <<-EOF > /usr/bin/graphdb_backup
+#!/bin/bash
+
+set -euxo pipefail
+
+az login --identity
+
+BACKUP_NAME="\$(date +'%Y-%m-%d_%H-%M-%S').tar"
+TEMP_BACKUP_DIR="/var/opt/graphdb/"
+RESOURCE_GROUP="\$(az vmss list --query "[0].resourceGroup" --output tsv)"
+STORAGE_ACCOUNT="\$(az storage account list --resource-group "\$RESOURCE_GROUP" --query "[].name" --output tsv)"
+CONTAINER_NAME="\$(az storage container list --account-name "\$STORAGE_ACCOUNT" --query "[].name" --output tsv --auth-mode login)"
+BLOB_URL="\$(az storage blob url --account-name "\$STORAGE_ACCOUNT" --container-name "\$CONTAINER_NAME" --name "\$BACKUP_NAME" --auth-mode login --output tsv)"
+
+function trigger_backup {
+  curl -X POST --output "\$${TEMP_BACKUP_DIR}\$${BACKUP_NAME}" -H 'Content-Type: application/json' 'http://localhost:7200/rest/recovery/backup'
```

Comment:
can't we pipe directly to `az`?

Reply:
We could, but I think this way it's more readable.

Reply:
Could do it in another PR, let's move forward for now.